### DISCUSSION TITLE
"forge test -m" not working in terminal?

### DISCUSSION THREAD

**Discussion by rohit-bhalerao-7** [Cyfrin/foundry-full-course-cu#112] [Q&A] [answered]
Hello guys, for me `forge test -m` wasn't working, throwing an error of unexpected argument '-m',  you can instead use `forge test --match-test <function>` which worked for me! I hope it helps. Patrick said we use -m to do test only on single function of our choice

**Comment by palmcivet7** [comment, marked as answer]
You can also use `forge test --mt <function>`

**Reply by rohit-bhalerao-7** [reply]
Thanks, mate! I'll use this one as it is convenient.

**Reply by cromewar** [reply]
Thanks @palmcivet7

**Comment by S23Web3** [comment]
Hi,

i still get it even with the mt


forge test -m functionName
error: unexpected argument '-m' found

Usage: forge test [OPTIONS]

For more information, try '--help'.
